spec/requests/api/v1/course_spec.rb

```diff
@@ -128,22 +128,22 @@ def get_course_codes_from_body(body)
 
       it "includes correct next link in response headers" do
         timestamp_of_first_course = 10.minutes.ago
-        Timecop.freeze(timestamp_of_first_course) do
-          first_course = create(:course,
-                                :infer_level,
-                                course_code: "LAST1",
-                                provider: provider)
+        first_course = create(:course,
```

Comment:
mmm ... are the request specs necessary? If so I would imagine something similar will be required for v2, v3 and public api request specs

Reply:
We are only really supporting the update_since stuff in /v3 and /public/v1 so we definitely don't need to update this version's spec. The existing specs for the other versions should be fine as is I would have thought 🤷

Reply:
These specs were failing after the change, so its just a re-order of object creation outside of Timecop so they pass (we have a unique index on `changed_at`).
I can remove them altogether if they are obsolete?

Reply:
Whenever `Apply` kicks in then this will be obsolete